src/utils/viewer/viewer.cpp

```diff
@@ -318,6 +319,14 @@ Key Commands:
   std::unique_ptr<ObjectPickingHelper> objectPickingHelper_;
   // returns the number of visible drawables (meshVisualizer drawables are not
   // included)
+
+  // Profiling
+  Mn::DebugTools::GLFrameProfiler::Values profilerValues =
+      Mn::DebugTools::GLFrameProfiler::Value::FrameTime |
+      Mn::DebugTools::GLFrameProfiler::Value::CpuDuration |
+      Mn::DebugTools::GLFrameProfiler::Value::GpuDuration;
+
+  Mn::DebugTools::GLFrameProfiler profiler_{profilerValues, 50};
```

Comment:
To avoid redundancy in the setup, you can also just default-construct the `profiler_` here and move the `profilerValues` to the constructor.